frontend/src/modules/S3_Datasets/components/DatasetUpload.js

```diff
@@ -153,7 +153,7 @@ export const DatasetUpload = (props) => {
               <TextField
                 fullWidth
                 helperText="Prefix without trailing slash at the end (e.g raw not raw/)"
-                label={`s3://${dataset.S3BucketName}/`}
+                label={`s3://${dataset.restricted.S3BucketName}/`}
```

Comment:
nit: i know tab is limited to admin but any need to also protect this with `emptyPrintUnauthorized` or similar?

Reply:
This PR has gone back and forth. At the beginning I added checks everywhere, but finally I opted to use it only when it will be used. The same as if in the backend an if condition is not needed then we don't add it